Source/OIDTokenUtilities.m

```diff
@@ -63,4 +69,21 @@ + (NSString *)redact:(NSString *)inputString {
   }
 }
 
++ (NSString*)formUrlEncode:(NSString*)inputString {
```

Comment:
Is it the right location to put this method?

Reply:
Works for me.